taglib/tagunion.cpp

```diff
@@ -199,12 +195,5 @@ void TagUnion::setTrack(unsigned int i)
 
 bool TagUnion::isEmpty() const
 {
-  if(d->tags[0] && !d->tags[0]->isEmpty())
-    return false;
-  if(d->tags[1] && !d->tags[1]->isEmpty())
-    return false;
-  if(d->tags[2] && !d->tags[2]->isEmpty())
-    return false;
-
-  return true;
+  return std::none_of(d->tags.begin(), d->tags.end(), [](auto tag) { return tag && !tag->isEmpty(); });
```

Comment:
```suggestion
  return std::none_of(d->tags.begin(), d->tags.end(),
    [](auto tag) { return tag && !tag->isEmpty(); });
```
The line is longer than 100 characters.

Reply:
Fixed.